doc/intl/fr/README.md

```diff
@@ -43,20 +45,25 @@ Principales ressources de développement :
 
 * [GitHub Discussion](https://github.com/dogecoin/dogecoin/discussions) est utilisé pour
   discuter des caractéristiques, planifiées et non planifiées, liées à la fois au développement du logiciel Dogecoin Core, les protocoles sous-jacents et l'actif DOGE.
-  
+
 * [Dogecoindev subreddit](https://www.reddit.com/r/dogecoindev/)
 
 ### Stratégie de version
+
 Les numéros de version suivent la sémantique ```major.minor.patch```.
 
 ### Branches
-Il y a 3 types de branches dans ce référentiel :
 
-- **master:** Stable, contient la dernière version de la dernière *major.minor* version.
-- **maintenance:** Stable, contient la dernière version des versions précédentes, qui sont toujours en maintenance active. Format : ```<version>-maint```
-- **development:** Instable, contient du nouveau code pour les versions prévues. Format : ```<version>-dev```
+Il y a 4 types de branches dans ce référentiel :
```

Comment:
"Dans ce référentiel" is the literal translation of "in this repository" but it's not grammatically correct. I would suggest phrasing this as follow:
```suggestion
Dans ce projet se trouvent 4 types de branches :
```
or even
```suggestion
Il y a 4 types de branches dans ce projet
```

Référentiel is a french word for reference. It has no direct link to a repository and a native french speaker would not think of "référentiel" as being a repository. Reading this would actually correctly translate to: "There are 4 types of branches under these references :"

Reply:
That is very strange. I doubt any native French speaker would find a similarity between Référentiel and Repository. Is it possible GitHub used AI or Google-like-translate to translate some of its features? Or maybe someone on a Crowdin-like-platform used a Google-like-translate to put in that word?

When I entered Repository on Google translate I got back Référentiel so it would make sense in my mind if that's how the word got there in the first place.

I find project to be a more visual word for repository than référentiel and also simpler to understand. Let's leave it up to the reviewers to decide which one they like more I suppose.

Reply:
@daank-c Look at how they translated "Pull requests" as "Demande de tirage". This makes absolutely no sense and is definitely not the correct translation. I suppose an online translator or the use of AI. The docs should definitely not use Référentiel.

Reply:
Repository would be translated to répertoire which would work. "Référentiel" is by definition a system of reference, which is what a repository is. So if the word "référentiel" doesn't sound right, I guess Répertoire would be good too.

Reply:
I think Répertoire would actually be more suited to refer to a directory.